memory/StackArray.h

```diff
@@ -7,9 +7,12 @@ template <typename T, size_t SIZE>
 class StackArray
 {
 public:
-    explicit StackArray(size_t size) : _size(size)
+    typedef const T* const_iterator;
+    typedef T* iterator;
+
+    StackArray() : _capacity(SIZE), _size(0), _dataPtr(_data) {}
```

Comment:
Yes, it is a bit like std:array but with ability to be instantiated with dynamic memory if needed.
Originally we used it when handling inbound sctp messages, that are usually small enough to be handled on stack within a certain limit. If it would be bigger occasionally, the heap is used.